pkg/controller/jaeger/jaeger_controller_test.go

```diff
@@ -191,7 +191,7 @@ func TestSetOwnerOnNewInstance(t *testing.T) {
 	persisted := &v1.Jaeger{}
 	cl.Get(context.Background(), req.NamespacedName, persisted)
 	assert.NotNil(t, persisted.Labels)
-	assert.Equal(t, "my-identity", persisted.Labels["app.kubernetes.io/managed-by"])
+	assert.Equal(t, "my-identity", persisted.Labels["jaegertracing.io/operated-by"])
```

Comment:
Change to use `v1.LabelOperatedBy` as well.

Reply:
Originally thought to add this as a string, to make sure we don't break clients using that, but on a second thought, this is not part of our public API, so, other people shouldn't be using it.

Reply:
Done